src/gluonts/dataset/common.py

```diff
@@ -207,13 +213,144 @@ def files(self) -> List[Path]:
         List[Path]
             List of the paths of all files composing the dataset.
         """
-        return util.find_files(self.path, self.is_valid)
+        return util.find_files(self.paths, self.is_valid)
 
     @classmethod
     def is_valid(cls, path: Path) -> bool:
-        # TODO: given that we only support json, should we also filter json
-        # TODO: in the extension?
-        return not (path.name.startswith(".") or path.name == "_SUCCESS")
+        valid_endings = [
+            ".json",
+            ".jsonl",
+            ".json.gz",
+            ".jsonl.gz",
+        ]
+        if not any(path.name.endswith(ending) for ending in valid_endings):
```

Comment:
```suggestion
        if not path.suffix in valid_endings:
```

Reply:
Would not work for `.json.gz`.

Reply:
Oh, of course! There is `.suffixes` which would work, but not sure it reads much nicer than this.